Dockerfile

```diff
@@ -1,17 +1,17 @@
-ARG GO_VERSION=1.12.10
+ARG GO_VERSION=1.13.0
 
 # OS-X SDK parameters
 # NOTE: when changing version here, make sure to also change OSX_CODENAME below to match
-ARG OSX_SDK=MacOSX10.10.sdk
-ARG OSX_SDK_SUM=631b4144c6bf75bf7a4d480d685a9b5bda10ee8d03dbf0db829391e2ef858789
+ARG OSX_SDK=MacOSX10.11.sdk
+ARG OSX_SDK_SUM=98cdd56e0f6c1f9e1af25e11dd93d2e7d306a4aa50430a2bc6bc083ac67efbb8
 
-# OSX-cross parameters. Go 1.11 requires OSX >= 10.10
-ARG OSX_VERSION_MIN=10.10
+# OSX-cross parameters. Go 1.13 requires OSX >= 10.11
+ARG OSX_VERSION_MIN=10.11
 ARG OSX_CROSS_COMMIT=a9317c18a3a457ca0a657f08cc4d0d43c6cf8953
```

Comment:
If we need to update to master I have already done it. https://github.com/tao12345666333/golang-cross/commit/afc6f85b4329acb8bb2f906a6b082f0bd840fb8b  

The CI has been passed. https://gitlab.com/taobeier/golang-cross/-/jobs/309376124